pyproject.toml

```diff
@@ -1,53 +1,76 @@
-[tool.poetry]
+[build-system]
+requires = ["poetry-core>=2.0.0,<3.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[project]
 name = "asynch"
 version = "0.3.0"
 description = "An asyncio driver for ClickHouse with native TCP support"
-authors = ["long2ice <long2ice@gmail.com>"]
-license = "Apache-2.0"
+authors = [
+    {name = "long2ice",email = "long2ice@gmail.com"}
+]
+license = {file = "LICENSE"}
 readme = "README.md"
+keywords = ["asyncio", "clickhouse", "python", "driver"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Framework :: AsyncIO",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
+    "Topic :: Database",
+    "Topic :: Software Development :: Libraries :: Python Modules"
+]
+requires-python = ">=3.9,<4.0"
+dependencies = [
+    "ciso8601 (>=2.3.2,<3.0.0)",
+    "leb128 (>=1.0.8,<2.0.0)",
+    "lz4 (>=4.4.3,<5.0.0)",
+    "pytz (>=2025.1,<2026.0)",
+    "tzlocal (>=5.3.1,<6.0.0)",
+    "zstd (>=1.5.6.6,<2.0.0.0)",
+]
+
+[project.optional-dependencies]
+compression = ["clickhouse-cityhash (>=1.0.2.4,<2.0.0.0)"]
+
+[project.urls]
 homepage = "https://github.com/long2ice/asynch"
 repository = "https://github.com/long2ice/asynch.git"
 documentation = "https://github.com/long2ice/asynch"
-keywords = ["asyncio", "clickhouse", "python", "driver"]
+
+[tool.poetry]
+package-mode = true
+requires-poetry = ">=2.1"
+include = ["LICENSE", "README.md"]
```

Comment:
Or should the `CHANGELOG.md` file be included anyway?

Reply:
Not sure why it shouldn't

Reply:
https://github.com/long2ice/asynch/pull/133/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L16